packages/css-utilities/README.md

```diff
@@ -0,0 +1,77 @@
+# `@shopify/css-utilities`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fcss-utilities.svg)](https://badge.fury.io/js/%40shopify%2Fcss-utilities.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/css-utilities.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/css-utilities.svg)
+
+A set of css styling related utilities.
+
+## Installation
+
+```bash
+$ yarn add @shopify/css-utilities
+```
+
+Both `classnames` and `variationName` are helper functions that are use to generate classNames.
```

Comment:
Still should be classNames for the first one, and maybe classname for the second one?